cmd/plugin/commands/routes/routes.go

```diff
@@ -0,0 +1,129 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package routes
+
+import (
+	"strings"
+
+	"github.com/apache/apisix-ingress-controller/cmd/plugin/common"
+	"github.com/apache/apisix-ingress-controller/cmd/plugin/kubectl"
+	"github.com/apache/apisix-ingress-controller/pkg/log"
```

Comment:
Please sort the imports, we should put the 3rd parties in the middle, and put our own packages at the bottom.

Reply:
beacuse the project not used sort the imports.